authors/apps/authentication/views.py

```diff
@@ -24,10 +58,47 @@ def post(self, request):
         # your own work later on. Get familiar with it.
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
-        serializer.save()
-
+        email = serializer.validated_data.get('email', None)
+        RegistrationAPIView.send_link(self,email)
+        serializer.save()        
         return Response(serializer.data, status=status.HTTP_201_CREATED)
 
+class VerifyAPIView(generics.CreateAPIView):
+    serializer_class = UserSerializer
```

Comment:
Do you need this Serializer? Doesn't look like you are serializing anything

Reply:
If I omit the serializer, I get an assertion error, I suppose when one is using generic views, you are required to have a serializer...not sure about this though